src/index.html

```diff
@@ -159,7 +165,7 @@ <h1 class="profile__title" id="profile-title">Jacques Cousteau</h1>
           type="button"
           id="avatar-edit-button"
         ></button>
-        <form class="modal__form profile__avatar-form">
+        <form class="modal__form profile__avatar-modal">

```

Comment:
`profile__avatar-modal`

There is a BEM mistake

Please, delete it and add an `id` for searching it


```js
        <form class="modal__form" id="avatar-form">


```